src/components/Nav/Nav.js

```diff
@@ -2,27 +2,27 @@ import React from 'react';
 import './Nav.css';
 import { Link } from "react-router-dom";
 
-const Nav = () => {
-    return (
-      <nav>
-        <ul>
-          <Link to="/">
-            <li>Home</li>
+const Nav = ({handleClick}) => {
+  return (
+    <nav>
+      <ul>
+        <Link to="/">
+          <li onClick={() => handleClick()}>Home</li>
+        </Link>
+        <div className="categoryNavList">
+          <Link to="/eco">
```

Comment:
I can't believe that bug was all due to a small missing syntax error! I'm glad we asked for help!